File: mobile/src/main/java/com/siliconlabs/bledemo/Browser/DebugModeCallback.java
package com.siliconlabs.bledemo.Browser;

import com.siliconlabs.bledemo.Bluetooth.BLE.BluetoothDeviceInfo;

public interface DebugModeCallback {

    void connectToDevice(BluetoothDeviceInfo device);

    void addToFavorite(String deviceAddress);

    void removeFromFavorite(String deviceAddress);

    void addToTemporaryFavorites(String deviceAddress);

    void updateCountOfConnectedDevices();
}

File: mobile/src/main/java/com/siliconlabs/bledemo/Browser/DebugModeCallback.kt
package com.siliconlabs.bledemo.Browser

import android.bluetooth.BluetoothDevice

interface DebugModeCallback {
    fun connectToDevice(device: BluetoothDevice)
    fun addToFavorite(deviceAddress: String)
    fun removeFromFavorite(deviceAddress: String)
    fun addToTemporaryFavorites(deviceAddress: String)
    fun updateCountOfConnectedDevices()
}
